### DISCUSSION TITLE
禁用了gridControl组件后, 引擎中的网格依旧存在

### DISCUSSION THREAD

**Discussion by guoqi-z** [galacean/discussion#95] [Engine Toolkit] [answered]
在代码中, 获取gridControl, 然后将其禁用后, 引擎中的gridControl依旧显示
![2e087376e401df79777337265d1e593](https://github.com/galacean/discussion/assets/31555729/fa4ad8fd-caf9-403c-baaa-68f5a930dcb0)
![975ae042944222c1645332f8db4e42b](https://github.com/galacean/discussion/assets/31555729/0701f4a0-1f4e-44ed-9db5-5aeb00f05f03)

**Comment by JujieX** [comment, marked as answer]
目前gridControl关闭模式是把entity置为false

**Reply by guoqi-z** [reply]
感谢, 这次按照你的说法, 总算是成功关闭了grid网格效果了.  
![image](https://github.com/galacean/discussion/assets/31555729/57fc6ba0-daa5-43ac-8053-2111a2ef6037)

**Reply by guoqi-z** [reply]
不过还有另外一个问题, 我之前是使用 gridComponent.camera.farClipPlane = 10000000; 将裁剪面设置到非常大的值, 来取消grid的显示的, 不知道这算不算是一个bug呢?

**Reply by JujieX** [reply]
是个bug，感谢提出
